### PR TITLE
Doc:translate of vela cli image docs

### PR DESCRIPTION
**Pull request by dazuimao1990**
translate of vela cli image docs to chinese. #807

### CODE REVIEW

**Comment by wonderflow** [comment]
please sign the DCO

**Comment by dazuimao1990** [comment]
Signed-off-by: dazuimao1990 gx_927@163.com

**Comment by dazuimao1990** [comment]
> please sign the DCO

Done

**Comment by wonderflow** [comment]
You should sign in the commit.

https://kubevela.net/docs/contributor/code-contribute#pass-all-the-ci-checks

**Comment by dazuimao1990** [comment]
> You should sign in the commit.
> 
> https://kubevela.net/docs/contributor/code-contribute#pass-all-the-ci-checks

The CI of the DCO looks like it passed, review it please.

File: i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/vela-cli-image.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: 用容器运行 vela 命令
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/vela-cli-image.md]:
```suggestion
title: 用容器镜像运行 vela 命令行
```

File: i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/vela-cli-image.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+---
+
+当前主题，用于描述如何利用 Docker 容器运行、配置 Vela 命令行工具，以及对其进行版本控制。有关更多使用 Docker 的信息，请参考 [Docker 官方文档](https://docs.docker.com/) 。
+
+由 KubeVela 社区官方支持并维护的 Docker 镜像保障了隔离性、可移植性以及安全性。这种免安装的使用方式允许你在一个基于容器的环境中使用 KubeVela 的命令行工具。
+
+## 前提条件
+
+你必须已经安装好了 Docker 。 如果你不知道如何安装它，请参考 [Docker 安装文档](https://docs.docker.com/install/) 。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/vela-cli-image.md]:
```suggestion
你必须已经安装好了 Docker 。如果你不知道如何安装它，请参考 [Docker 安装文档](https://docs.docker.com/install/) 。
```

**Review by wonderflow — APPROVED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/vela-cli-image.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    $ docker run --rm -it -v ~/.kube:/root/.kube oamdev/vela-cli version
+    CLI Version: master
+    Core Version:
+    GitRevision: git-1d823780
+    GolangVersion: go1.17.10
+    ```
+
+* `--rm` – 指定该参数，将会在退出容器的同时删除它。
+
+* `-it` – 指定该参数，将会打开一个伴随标准输入的伪TTY。这将允许你为运行中的 KubeVela 命令行容器提供输入，例如，使用 `vela port-forward` 命令。
```
Comment (charlie0129) [inline i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/vela-cli-image.md]:
```suggestion
* `-it` – 指定该参数，将会打开一个伴随标准输入的伪 TTY。这将允许你为运行中的 KubeVela 命令行容器提供输入，例如，使用 `vela port-forward` 命令。
```

**Pull request merged**